tools/cli/adminCommands.go

```diff
@@ -73,7 +73,7 @@ func AdminShowWorkflow(c *cli.Context) {
 	serializer := serialization.NewSerializer()
 	var history []*commonpb.DataBlob
 	if len(tid) != 0 {
-		histV2 := cassandra.NewHistoryV2PersistenceFromSession(session, log.NewNoopLogger())
+		histV2 := cassandra.NewWorkflowStore(-1, session, log.NewNoopLogger())
```

Comment:
this is weird, can we try to put shard ID as part of request? instead of internal var of execution / workflow store

Reply:
Updated. This is no longer needed.